public/en/releases.html

```diff
@@ -0,0 +1,445 @@
+<!DOCTYPE html>
```

Comment:
Great. FYI I was referring to a PR against the master branch not specifically this open PR. But I think that made sense ;)

Good call. Uppercase it does look weird...